modules/n1ql/pages/n1ql-language-reference/explainfunction.adoc

```diff
@@ -0,0 +1,109 @@
+= EXPLAIN FUNCTION
+:description: EXPLAIN FUNCTION provides information about the execution plan for the {sqlpp} subqueries or embedded statements defined for a specified user-defined function. 
```

Comment:
Could we modify the introduction to imply that this returns query plans for SQL++ within/ inside the user-defined function definition ?

Reply:
@dhanyagowrish I thought that "subqueries" and "embedded statements" covered this idea, would you be able to rephrase more accurately or so that it is easier to understand?

Here's my attempt to put the focus more on UDF: 

For a specified user-defined function, you use EXPLAIN FUNCTION to expose the execution plan of its {sqlpp} subqueries or embedded statements.

Reply:
What I meant was that, it would be ideal to imply that EXPLAIN FUNCTION returns query plans for statements ( whether subqueries or embedded statements ) within/ inside the UDF definition.

Reply:
@simon-dew can you suggest phrasing for the description to accommodate Dhanya's request?

Reply:
I would have said "defined _within_ a specified user-defined function", but your new phrasing works just as well